Makefile

```diff
@@ -105,6 +105,23 @@ build-release-tarballs: ## Create a series of `.tar.gz` files in the BIN_DIR dir
 
 ##@ Test
 
+UNIT_TEST_ARGS := --locked --workspace --all-features -E 'kind(lib)' -E 'kind(bin)' -E 'kind(proc-macro)'
+COV_FILE := lcov.info
+
+.PHONY: test-unit
+test-unit: ## Run unit tests.
+	cargo nextest run $(UNIT_TEST_ARGS)
+
+.PHONY: cov-unit
+cov-unit: ## Run unit tests with coverage.
+	rm -f $(COV_FILE)
+	cargo llvm-cov nextest --lcov --output-path $(COV_FILE) $(UNIT_TEST_ARGS)
```

Comment:
I'd prefer to use `cargo llvm-cov nextest --html $(UNIT_TEST_ARGS)` directly there and only use `open target/llvm-cov/html/index.html` in `cov-report-html` but I don't know what you guys prefer.

Reply:
fine w/ me either way - up to you